oam/install_scripts/columnstore-pre-uninstall.in

```diff
@@ -118,6 +115,9 @@ rm -f @ENGINE_SYSCONFDIR@/columnstore/AlarmConfig.xml.installSave
 #remove OAMdbrootCheck file
 rm -f /var/lib/columnstore/data*/OAMdbrootCheck > /dev/null 2>&1
 
+rm -f /etc/rsyslog.d/49-columnstore.conf > /dev/null 2>&1
+rm -rf /tmp/columnstore_tmp_files > /dev/null 2>&1
```

Comment:
I don't think we need the latter for a final shutdown.

Reply:
Just realized, this is duplicate code to line 49. Remove one of these, most likely 49